rts/Lua/LuaHandle.cpp

```diff
@@ -1614,6 +1615,8 @@ void CLuaHandle::Update()
 
 	// call the routine
 	RunCallIn(L, cmdStr, 0, 0);
+
+	LuaVFSDownload::ProcessDownloads();
```

Comment:
its meant to be called just once every Update event. not sure how often/when LuaHandle::Update is called.